src/main/java/org/opensearch/plugin/insights/core/listener/QueryInsightsListener.java

```diff
@@ -54,6 +56,8 @@ public final class QueryInsightsListener extends SearchRequestOperationsListener
 
     private final QueryInsightsService queryInsightsService;
     private final ClusterService clusterService;
+    private boolean groupingFieldNameEnabled;
+    private boolean groupingFieldTypeEnabled;
```

Comment:
Where do we read this? do we need a follow up PR to use it?

Reply:
Yup @dzane17 's PR will use this